apisix/plugins/proxy-rewrite.lua

```diff
@@ -70,8 +74,58 @@ local schema = {
         },
         headers = {
             description = "new headers for request",
-            type = "object",
-            minProperties = 1,
+            anyOf = {
```

Comment:
We should use `oneOf`?

Reply:
No, we should use anyOf to maintain downward compatibility. The response-rewrite plugin has the same schema.

Reply:
`anyOf` means you can configure both the old way and the new way at the same time, which is unnecessary. While `oneOf` means you can configure either the old way or the new way.

Reply:
If replace `anyOf` with `oneOf`, the schema can't work correctly.

Reply:
Done